src/VssCommandProcessor.cpp

```diff
@@ -498,7 +498,7 @@ string VssCommandProcessor::processQuery(const string &req_json,
         return JsonResponses::malFormedRequest("Unknown action requested");
       }
     }
-  } catch (jsoncons::json_parse_exception &e) {
+  } catch (jsoncons::ser_error &e) {
     logger->Log(LogLevel::ERROR, "JSON parse error");
```

Comment:
not only parse error, right?

Reply:
json_parse_exception has been replaced, see https://github.com/danielaparker/jsoncons/blob/5be801990f996f9905e8a3d442f3810ae21c74de/include/jsoncons/json_exception.hpp#L172